pkg/polaris-reporting/creater.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Synopsys, Inc.
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package polarisreporting
+
+import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// GetPolarisReportingSecrets ...
+func GetPolarisReportingSecrets(namespace, GCPServiceAccount string) (map[string]runtime.Object, error) {
```

Comment:
Can we de-dup and combine this with: https://github.com/blackducksoftware/synopsys-operator/blob/master/pkg/polaris/creater.go#L160

Reply:
The Polaris function returns two other secrets that Reporting doesn't need. And the secret it does return as the wrong name. We could refactor the Polaris one so that Reporting could use it more easily. I will make a TODO to implement that for the next release (don't want to break Polaris code by accident for this release)